CITATION.cff

```diff
@@ -0,0 +1,61 @@
+# Pleace cite the following article when referencing Celeritas in a publication:
+#
+# (Plain text, Chicago author-date 17th ed):
+#   Johnson, Seth R., Amanda Lund, Philippe Canal, Stefano C. Tognini, Julien
+#   Esseiva, Soon Yung Jun, Guilherme Lima, et al. 2024. “Celeritas:
+#   Accelerating Geant4 with GPUs.” EPJ Web of Conferences 295:11005.
+#   https://doi.org/10.1051/epjconf/202429511005.
+#
+# (BibLaTeX):
+#    @article{celer-chep-2024,
+#      title = {Celeritas: {{Accelerating Geant4}} with {{GPUs}}},
+#      author = {Johnson, Seth R. and Lund, Amanda and Canal, Philippe and Tognini, Stefano C. and Esseiva, Julien and Jun, Soon Yung and Lima, Guilherme and Biondo, Elliott and Evans, Thomas and Demarteau, Marcel and Romano, Paul},
+#      date = {2024},
+#      journaltitle = {EPJ Web of Conferences},
+#      volume = {295},
+#      pages = {11005},
+#      doi = {10.1051/epjconf/202429511005}
+#    }
+#
+# (CFF):
+
+cff-version: 1.2.0
+type: software
+message: "Please cite the following article when referencing Celeritas in a publication."
+title: "Celeritas: Accelerating Geant4 with GPUs"
+abstract: >-
+  Celeritas [1] is a new Monte Carlo (MC) detector simulation code designed for computationally intensive applications (specifically, High Luminosity Large Hadron Collider (HL-LHC) simulation) on high-performance heterogeneous architectures. In the past two years Celeritas has advanced from prototyping a GPU-based single physics model in infinite medium to implementing a full set of electromagnetic (EM) physics processes in complex geometries. The current release of Celeritas, version 0.3, has incorporated full device-based navigation, an event loop in the presence of magnetic fields, and detector hit scoring. New functionality incorporates a scheduler to offload electromagnetic physics to the GPU within a Geant4-driven simulation, enabling integration of Celeritas into high energy physics (HEP) experimental frameworks such as CMSSW. On the Summit supercomputer, Celeritas performs EM physics between 6× and 32× faster using the machine’s Nvidia GPUs compared to using only CPUs. When running a multithreaded Geant4 ATLAS test beam application with full hadronic physics, using Celeritas to accelerate the EM physics results in an overall simulation speedup of 1.8–2.3× on GPU and 1.2× on CPU.
+preferred-citation:
```

Comment:
In the "Citing Celeritas" section in the README we point to the [DOECode](https://www.osti.gov/doecode/biblio/94866) registration. Do we want to make sure we're consistent with what we ask for there and our preferred citation here?

Reply:
Oh good point. I'll add the DOE code as the main "reference" and then update both to the "citable" reference.